src/sdl2_wrapper.cpp

```diff
@@ -197,6 +202,42 @@ CAMLprim value resdl_SDL_GetNativeWindow(value vWin) {
   CAMLreturn((value)pNativeWindow);
 };
 
+CAMLprim value resdl_SDL_SetMacTitlebarTransparent(value vWin) {
+  CAMLparam1(vWin);
+
+  #ifdef __APPLE__
+  SDL_Window *win = (SDL_Window *)vWin;
+  SDL_SysWMinfo wmInfo;
+  SDL_VERSION(&wmInfo.version);
+  SDL_GetWindowWMInfo(win, &wmInfo);
+  NSWindow *nWindow = wmInfo.info.cocoa.window;
+  [nWindow setStyleMask: [nWindow styleMask] | NSWindowStyleMaskFullSizeContentView];
+  [nWindow setTitlebarAppearsTransparent:YES];
+  #endif
+
+  CAMLreturn(Val_unit);
+}
+
+CAMLprim value resdl_SDL_SetMacBackgroundColor(value vWin, value r, value g, value b, value a) {
```

Comment:
With the change to use `NSWindowStyleMaskFullSizeContentView` - is there still a use case for `resdl_SDL_SetMacBackgroundColor`?

Reply:
Yeah we can probably get rid of this now. It might be nice to have though -- up to you.

Reply:
Hmm, I guess it can be used separately - for example, you could call `setMacBackgroundColor` w/o needing `setMacTitlebarTransparent`.

And I realized this can set the default color prior to rendering for the window background, beyond the titlebar - so that would be useful too. Let's keep it 👍

Reply:
Sounds good! 👍